proto/engine/v1/execution_engine.proto

```diff
@@ -244,6 +244,27 @@ message BlobsBundle {
   ];
 }
 
+message BlobsBundleV2 {
+  // The KZG commitments of the blobs.
+  repeated bytes kzg_commitments = 1 [
+    (ethereum.eth.ext.ssz_size) = "?,48",
+    (ethereum.eth.ext.ssz_max) = "max_blob_commitments.size"
+  ];
+
+  // The cell_proofs for all blobs
+  // TODO: update ssz_max when it's defined in the spec
+  repeated bytes cell_proofs = 2 [
+    (ethereum.eth.ext.ssz_size) = "?,48",
+    (ethereum.eth.ext.ssz_max) = "max_blob_commitments.size"
+  ];
+
+  // The blobs itself.
+  repeated bytes blobs = 3 [
+    (ethereum.eth.ext.ssz_size) = "?,blob.size",
+    (ethereum.eth.ext.ssz_max) = "max_blob_commitments.size"
+  ];
+}
+
```

Comment:
In general, it’s best to define new protobuf messages in their own files, such as `execution_engine_eip7594.proto` or something similar. Keeping everything in a single file increases the risk of conflicts, especially with multiple feature branches in development

Reply:
Good to know, will update